pkg/controller/registry/reconciler/grpc.go

```diff
@@ -12,12 +11,42 @@ import (
 	"github.com/sirupsen/logrus"
 	corev1 "k8s.io/api/core/v1"
 	v1 "k8s.io/api/core/v1"
+	kscheme "k8s.io/client-go/kubernetes/scheme"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/util/intstr"
+	"k8s.io/client-go/rest"
+	controllerclient "github.com/operator-framework/operator-lifecycle-manager/pkg/lib/controller-runtime/client"
+	k8scontrollerclient "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	CatalogSourceUpdateKey = "catalogsource.operators.coreos.com/update"
+	RegistryFieldManager = "olm.registry"
 )
+var (
+	scheme = runtime.NewScheme()
+	ssaClient *controllerclient.ServerSideApplier
```

Comment:
This should be dependency injected or constructed from a client that is dependency injected.